GauchoGrubAndroid/app/src/main/java/com/g10/gauchogrub/io/MenuParser.java

```diff
@@ -1,4 +1,80 @@
 package com.g10.gauchogrub.io;
 
-public class MenuParser {
-}
+import com.g10.gauchogrub.menu.DayMenu;
+import com.g10.gauchogrub.menu.Meal;
+import com.g10.gauchogrub.menu.MenuItem;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import java.util.logging.Logger;
+
+public class MenuParser{
+
+    public final static Logger logger = Logger.getLogger("MenuFragment");
+
+    public DayMenu getDayMenu(String menuString) {
```

Comment:
Confusing name. Should be something like: getMenuList() and should return a list of menus.